scanner/src/main/kotlin/experimental/ScannerWrapper.kt

```diff
@@ -54,6 +55,11 @@ sealed interface ScannerWrapper {
      * `options.ScanCode.criteria.minScannerVersion=3.0.2`.
      */
     val criteria: ScannerCriteria
+
+    /**
+     * Filter the scanner specific options to remove any secrets, like credentials.
+     */
+    fun filterSecretOptions(options: ScannerOptions) = options
```

Comment:
Shouldn't we align the naming to `Scanner.filterOptionsForResult()` to avoid confusion? After all, it's up to the scanner implementor what this filters... it does not necessarily need to be secrets only.

Reply:
We do not have any use case for filtering anything but secrets so I thought better be specific.

Reply:
But others may have other use-cases in the future, and then not dare to use this function because of its name... just thinking out aloud.

Reply:
Approving anyway.